engineering/teams/methodology.md

```diff
@@ -0,0 +1,135 @@
+
+# Internal Team methodology
+
+Date: 2018-29-11
```

Comment:
```suggestion
Date: 2018-11-29
```

Reply:
it's a moved content, so the date should be the old one.